tests/vcd_catalog_update.py

```diff
@@ -0,0 +1,25 @@
+import os
+import unittest
+import yaml
+from pyvcloud.vcd.client import BasicLoginCredentials
+from pyvcloud.vcd.client import Client
+from pyvcloud.vcd.org import Org
+from pyvcloud.vcd.test import TestCase
+
+class UpdateCatalog(TestCase):
+
+    def test_create_catalog(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, resource=logged_in_org)
+        catalog = org.create_catalog(self.config['vcd']['catalog'], 'test catalog')
+        assert self.config['vcd']['catalog'] == catalog.get('name')
+
+    def test_update_catalog(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, resource=logged_in_org)
+        catalog = org.update_catalog(self.config['vcd']['catalog'], self.config['vcd']['new_name'], self.config['vcd']['new_desc'])
```

Comment:
Line too long add a #  NOQA at the end of the line.

Reply:
You can probably make the line into 2. I think the line length limit is 80 chars.